go.mod

```diff
@@ -6,6 +6,9 @@ require (
 	github.com/BurntSushi/toml v0.4.1
 	github.com/onsi/gomega v1.17.0
 	github.com/paketo-buildpacks/occam v0.2.1
-	github.com/paketo-buildpacks/packit v1.3.1
+	github.com/paketo-buildpacks/packit/v2 v2.0.0
 	github.com/sclevine/spec v1.4.0
 )
+
+// TODO: Remove this when v0.32.0 comes out with CycloneDX JSON support
```

Comment:
We can remove this when packit v2.0.1 comes out later on.

Reply:
alternatively, remove the syft line and replace the packit line with:

```
replace github.com/paketo-buildpacks/packit/v2 => github.com/paketo-buildpacks/packit/v2 v2.0.1-0.20211209184112-5bda788cab1c
```